polkadot/node/core/backing/src/error.rs

```diff
@@ -105,6 +105,9 @@ pub enum Error {
 
 	#[error("Availability store error")]
 	StoreAvailableData(#[source] StoreAvailableDataError),
+
+	#[error("Data is not available")]
+	DataNotAvailable,
```

Comment:
This doesn't say anything

Reply:
For a temporary solution, I have changed the name and description of the error but, as discussed separately, will try to replace RuntimeApiErrors with Subsystem errors in a separate PR.